_tour/implicit-parameters.md

```diff
@@ -13,52 +13,58 @@ previous-page: self-types
 redirect_from: "/tutorials/tour/implicit-parameters.html"
 ---
 
-A method with _implicit parameters_ can be applied to arguments just like a normal method. In this case the implicit label has no effect. However, if such a method misses arguments for its implicit parameters, such arguments will be automatically provided.
+A method can have an _implicit_ parameter list, marked by the _implicit_ keyword at the start of the parameter list. If the parameters in that parameter list are not passed as usual, scala will look if it can get an implicit value of the correct type, and if it can, pass it automatically.
 
-The actual arguments that are eligible to be passed to an implicit parameter fall into two categories:
+The places scala will look for these parameters fall into two categories:
 
-* First, eligible are all identifiers x that can be accessed at the point of the method call without a prefix and that denote an implicit definition or an implicit parameter.
-* Second, eligible are also all members of companion modules of the implicit parameter's type that are labeled implicit.
+* First, eligible are all identifiers that can be accessed at the point of the method call without a prefix and that denote an implicit definition or an implicit parameter.
+* Second, eligible are also all members of companion objects of the implicit parameter's type that are labeled implicit.
```

Comment:
I know it was like this, but the phrasing "eligible are all identifiers that ..." sounds like a 19th century legal text. 

Why not something like...

> ... two categories:
> - Implicit definitions and implicit parameters that can be referenced at the point of the method call without a prefix, and
> - Members of companion objects of the implicit parameter's type that are marked implicit.

This elides the "parts" bit from the spec but I think that is needlessly complicated to mention in the tour.

Reply:
The swing I took at this also addresses the two below comments. WDYT?